Dockerfile

```diff
@@ -30,4 +35,4 @@ RUN chgrp -R 0 $WORKDIR && \
 
 EXPOSE 3000
 
-CMD ["entrypoint"]
+ENTRYPOINT ["entrypoint"]
```

Comment:
Slightly off topic, but I'm running into the issue with my #58 PR - can we add https://github.com/ManageIQ/topological_inventory-core/blob/c0d48753f041cac1d56df65b02c482eaedc97b0a/docker-assets/entrypoint#L5-L7 to the `entrypoint`?

Reply:
Let's add that to #58 for sure, since that's where v2_key and manageiq-password are being added and that particular line is coupled with that change,